packages/zoe/src/zoeService/zoeSeat.js

```diff
@@ -35,8 +35,17 @@ export const makeZoeSeatAdminKit = (
   // This does not suppress any error messages.
   payoutPromiseKit.promise.catch(_ => {});
   const { notifier, updater } = makeNotifierKit();
-
-  let currentAllocation = initialAllocation;
+  let actualProposal;
+  let proposalSupplied;
+  if (!proposal) {
+    proposalSupplied = false;
+    // enough to mollify typescript. uses should be guarded by !proposalSupplied
+    actualProposal = { give: null, want: null, exit: null };
+  } else {
+    proposalSupplied = true;
+    actualProposal = proposal;
+  }
+  let currentAllocation = harden({});
```

Comment:
You wouldn't have to mollify Typescript if:
```suggestion
  /** @type {ProposalRecord=} */
  let actualProposal;
  if (proposal) {
    actualProposal = proposal;
  }
  let currentAllocation = harden({});
```

and then test `actualProposal` instead of `proposalSupplied`.